src/controllers/scripting/legacy/controllerscriptinterfacelegacy.h

```diff
@@ -72,6 +86,13 @@ class ControllerScriptInterfaceLegacy : public QObject {
             const double rate = -10.0);
     Q_INVOKABLE void softStart(const int deck, bool activate, double factor = 1.0);
 
+    Q_INVOKABLE QByteArray convertCharset(
+            const ControllerScriptInterfaceLegacy::WellKnownCharsets
+                    targetCharset,
+            const QString& value);
+
+    Q_INVOKABLE QByteArray convertCharset(const QString& targetCharset, const QString& value);
```

Comment:
```suggestion
```
This API is unsafe and error prone! The number of supportet charsets must mit finite and tesable. This change was already requested in the orignial PR # 13772 and applies here as well!